src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -18,18 +21,22 @@ namespace Microsoft.CodeAnalysis.Editor.InlineParameterNameHints
     internal class InlineParameterNameHintsTag : IntraTextAdornmentTag
     {
         public const string TagId = "inline parameter name hints";
-
         /// <summary>
         /// Creates the UIElement on call
         /// Uses PositionAffinity.Predecessor because we want the tag to be associated with the preceding character
         /// </summary>
         /// <param name="text">The name of the parameter associated with the argument</param>
-        public InlineParameterNameHintsTag(string text, double lineHeight, TextFormattingRunProperties format)
-            : base(CreateElement(text, lineHeight, format), removalCallback: null, PositionAffinity.Predecessor)
+        public InlineParameterNameHintsTag(string text, IWpfTextView textView, TextFormattingRunProperties format)
+            : base(CreateElement(text, textView, format), removalCallback: AdornmentCallbackFunction, PositionAffinity.Predecessor)
+        {
+        }
+
+        private static void AdornmentCallbackFunction(object tag, UIElement element)
         {
+            //
         }
```

Comment:
what is this empty method for?

Reply:
Oops this was some debugging code that got left in.

Reply:
Sorry, forgot to remove it